central/debug/service/service.go

```diff
@@ -265,23 +265,23 @@ func addJSONToZip(zipWriter *zip.Writer, fileName string, jsonObj interface{}) e
 }
 
 func zipPrometheusMetrics(zipWriter *zip.Writer, name string) error {
-	metricsWriter, err := zipWriter.Create(name)
+	metricsWriter, err := zipWriterWithCurrentTimestamp(zipWriter, name)
 	if err != nil {
 		return err
```

Comment:
Let's be good citizens and  wrap the error similar to
https://github.com/stackrox/stackrox/pull/5231/files#diff-c81b773cf0c8cb97684ec44acb5cb687c550f73d5e24ea4df18d7251c8bcf185R258

Reply:
How about wrapping error at `zipWriterWithCurrentTimestamp` level so we will have full context and don't need to wrap in every method call?

Reply:
sounds good to me 👍